qa/multi-cluster-search/src/test/resources/rest-api-spec/test/remote_cluster/10_basic.yml

```diff
@@ -103,6 +105,19 @@
            - '{"index": {"_index": "test_index"}}'
            - '{"f1": "remote_cluster", "animal": "chicken", "filter_field": 0}'
 
+  - do:
+      bulk:
+        refresh: true
+        body:
+          - '{"index": {"_index": "field_caps_index_1"}}'
+          - '{"created_at": "2018-01-05"}'
+          - '{"index": {"_index": "field_caps_index_1"}}'
+          - '{"created_at": "2017-12-01"}'
+          - '{"index": {"_index": "field_caps_index_3"}}'
+          - '{"created_at": "2019-10-01"}'
+          - '{"index": {"_index": "field_caps_index_3"}}'
+          - '{"created_at": "2020-01-01"}'
```

Comment:
maybe we could even set a routing value on one of these two indices to force all documents to be in the same shard, which should exercise more the logic about iterating shards until one can match?

Reply:
++, I pushed https://github.com/elastic/elasticsearch/pull/57276/commits/fb3007c0832b7a5eb604e76978fb0047d70ecd61